src/batch/normalization/services/computeLabelStatus.ts

```diff
@@ -61,9 +61,9 @@ export function computeLabelStatus(decisionDto: DecisionTJDTO): LabelStatus {
   if (!isDecisionFromTJTransmissibleToCC(decisionDto.codeDecision)) {
     logger.error({
       ...formatLogs,
-      msg: 'Decision can not be treated by Judilibre because codeDecision is not in authorized codeDecision list, changing LabelStatus to ignored_codeDecisionNonTransmisCC.'
+      msg: 'Decision can not be treated by Judilibre because codeDecision is not in authorized codeDecision list, changing LabelStatus to ignored_codeDecisionBloqueCC.'
     })
```

Comment:
Amelioration :
Est-ce que pour éviter d'avoir a réécrire à chaque changement on se baserait sur la valeur de de dbsder-api-types

`Decision debat is not public, changing LabelStatus to ${LabelStatus.STATUT}.`